docs/deployment/resource-providers/standalone/index.md

```diff
@@ -24,153 +24,203 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This page provides instructions on how to run Flink in a *fully distributed fashion* on a *static* (but possibly heterogeneous) cluster.
```

Comment:
I don't like the "Overview" entry in the menu panel on the left as we're not giving an overview but introduce the basic (or native) deployment of a Standalone cluster. What is your opinion on the naming?

Reply:
I had exactly the same concern, however I didn't manage to rename it, without the navigation falling apart.